test/com/google/javascript/jscomp/integration/ProductionCoverageInstrumentationPassIntegrationTest.java

```diff
@@ -116,6 +121,32 @@ public void testNoTranspilationFunctionInstrumentation() {
     test(options, sourceArr, expectedArr);
   }
 
+  @Test
+  public void testParamMappingIsCreated() {
+    CompilerOptions options = createCompilerOptions();
+
+    String source = lines("function foo() { ", "   console.log('Hello'); ", "}");
```

Comment:
nit: add `//` to preserve line breaks here

lines(
    "function foo() { ", //
    "   console.log('Hello'); ",
    "}")

Reply:
Not sure what you mean by "add `//` to preserve line breaks here"? I did break it down for readability however
```
lines( 
   "function foo() {",
   ...
);
```